hphp/runtime/vm/jit/vasm-arm.cpp

```diff
@@ -602,6 +603,10 @@ void Vgen::emit(const andli& i) {
   a->And(W(i.d), W(i.s1), MSKTOP(i.s0.l()), SetFlags);
 }
 
+void Vgen::emit(const testbi& i) {
+  a->Tst(W(i.s1), i.s0.ub());
+}
+
 void Vgen::emit(const testli& i) {
   a->Tst(W(i.s1), MSKTOP(i.s0.l()));
 }
```

Comment:
Can you move these guys to be inline in the struct definition, like the other `test` instructions?